rr_iarrc/src/laplacian_line_detection/laplacian_line_detector.cpp

```diff
@@ -12,53 +12,58 @@
 #include <stdio.h>
 
 using namespace cv;
```

Comment:
Separate from this PR, it like to see us remove all "using namespace cv" invocations at some point in the future.